specification/search/data-plane/Microsoft.Azure.Search.Service/track1/preview/2019-05-06-preview/searchservice.json

```diff
@@ -1619,7 +1619,74 @@
       ],
       "x-ms-enum": {
         "name": "TokenizerName",
-        "modelAsString": false
+        "modelAsString": true,
```

Comment:
Do we have any extensible enums in searchindex.json?

Reply:
Yes we do, but I only focused in this PR on those models for which we have customizations... And we have no such instance in searchindex.json.
I don't intend to convert all "enums" to "extensible enums"....

Reply:
I meant specifically extensible enums, not any old enums. If there aren't any, that's fine.